accounting/invoice/models.py

```diff
@@ -332,7 +299,14 @@ def save(self, *args, **kwargs):
         if self.auto_create_pdf_on_save:
             self.to_pdf()
         if self.auto_upload_google_drive_on_save:
-            self.upload_to_google_drive()
+            file = self.upload_to_google_drive(
+                file_path=self.pdf_path,
+                target_path=[self.date.strftime('%Y'), 'invoices-in', self.date.strftime('%m')],
+                title=self.pdf_filename,
+            )
+            # We can't save this field because we're already in the save method, so set it and then update it directly.
```

Comment:
So should that need to happen *before* the `super().save()` method is called?

Reply:
I tried this and it fails miserably -- at this point I don't recall the error message. Either way I wouldn't bother with this: I'm deleting it in the very next PR that's up, #10.